bot/exts/fun/tic_tac_toe.py

```diff
@@ -192,41 +214,70 @@ def format_board(self) -> str:
     async def play(self) -> None:
```

Comment:
I don't have any concrete suggestions here, but this method is pretty complicated. I don't understand how the game logic works